### ISSUE TITLE
Training issure

### ISSUE THREAD

**Issue by michelleqyhqyh** [carboncoo/mask-align#4]
![training](https://user-images.githubusercontent.com/45490378/141267037-a7d4c324-dc55-40ff-9a7a-a257b972c69d.png)
There is an error during the training when I used my training data. However, the training steps didn't stopped. Do you know what it's going on with it?
